Holovibes/sources/compute/rendering.cc

```diff
@@ -227,10 +236,28 @@ void Rendering::insert_apply_contrast(WindowKind view)
         });
 }
 
-void Rendering::insert_compute_autocontrast(std::atomic<bool>& autocontrast_request,
-                                            std::atomic<bool>& autocontrast_slice_xz_request,
-                                            std::atomic<bool>& autocontrast_slice_yz_request,
-                                            std::atomic<bool>& autocontrast_filter2d_request)
+/*! \brief Tell if the contrast should be applied
+ *
+ * \param request The request for autocontrast
+ * \param queue The accumulation queue
+ * \return true if the contrast should be applied
+ */
+inline bool apply_contrast(bool request, const std::unique_ptr<Queue>& queue)
```

Comment:
Do not forget about [in] for doxgygen, also the inlined functions should be in the header file. I agree with Bastien that the name is unclear.

Reply:
Should the function even be inline ? it feels a bit weird having a more than 1 line inline function...

Reply:
Renamed the function to `should_apply_contrast` (inspired by the doxygen description, which contains the word 'should')